src/border.ts

```diff
@@ -32,31 +33,46 @@ const countryCodes: { [key in Country]: number } = {
 
 interface Options {
   /* Default country border to display */
-  defaultCountry?: Country;
+  country?: Country;
 }
 
 let isInitialized = false;
 
 /**
  * Select and switch borders point of view according to different countries definition.
  */
-export function border(map: mapboxgl.Map, options?: Options) {
-  init(map);
+export class Border extends Control {
+  _options?: Options;
 
-  if (options?.defaultCountry) {
-    setBorder(map, options.defaultCountry);
-  } else {
-    resetBorder(map);
+  constructor(options?: Options) {
+    super();
+    this._options = options;
   }
 
-  return {
-    resetBorder() {
-      resetBorder(map);
-    },
-    setBorder(country: Country) {
-      setBorder(map, country);
-    },
-  };
+  _initialUpdate() {
```

Comment:
You can use `private` and `protected` in TypeScript ;)

Reply:
Fixed

Reply:
👌